jsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
+	"compilerOptions": {
+		"baseUrl": ".",
+		"paths": {
+			"@wordpress/*": ["./*", "./packages/*/src"]
+		}
+	},
+	"exclude": [
+		"build",
+		"build-module",
+		"coverage",
+		"node_modules",
+		"test/e2e/test-plugins",
+		"vendor"
```

Comment:
I borrowed these excludes from `.eslintignore`—not sure how up to date they are. From what I understand reading [the docs](https://code.visualstudio.com/docs/languages/jsconfig), VS Code just recommends that you exclude directories with lots of irrelevant source files e.g. `build` and `node_modules`.